projects/php.net/package.yml

```diff
@@ -0,0 +1,38 @@
+distributable:
+  url: https://github.com/php/php-src/archive/refs/tags/php-{{ version }}.tar.gz
```

Comment:
the tagged tarballs typically do not contain `./configure` I'd looking for the official downloads.

Also recommend running `pkg build` locally before submitting as debug cycles will be way faster for you!

Finally: super glad to have you working on php. thank you.